src/components/text-editor/prosemirror-adapter/menu/menu-commands.ts

```diff
@@ -138,12 +173,52 @@ const createWrapInCommand = (
         throw new Error(`Node type "${nodeType}" not found in schema`);
     }
 
-    const command: CommandWithActive = wrapIn(type);
+    let command: CommandWithActive;
+    if (nodeType === EditorMenuTypes.Blockquote) {
+        command = toggleBlockType(schema, EditorMenuTypes.Blockquote, {}, true);
+    } else {
+        command = wrapIn(type);
+    }
+
     setActiveMethodForWrap(command, type);
 
     return command;
 };
 
+const toggleList = (listType) => {
```

Comment:
I'm not sure if markdown supports nested lists. The current behaviour seems to be fairly similar to how the list buttons work in the editor here on GitHub. In the future, I imagine we might want more advanced behaviour, at least when the output mode is html, but if so, that's for another time.

👍